.github/ISSUE_TEMPLATE/draft-suggestion.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Proposals suggestion
+about: Suggestion for improving existing proposals
+---
+
+## On proposal
+
+Which proposal is your suggestion intended for? (e.g. ACP, WAC+)
+
+## Details
+
+
+
+## Acceptance criteria
+
+What actions are needed to resolve this issue? (checklist)
+
+* [ ] 
```

Comment:
Strange that this didn't include the following as given, I wonder why:

[ ] Proposer will i) implement ii) share implementation publicly -- See also uc-survey.

Reply:
Flagging an issue does not equate to proposing anything, nor to having the ability, resources, etc., necessary to fix that issue (nor implement a proposal, if that is what the issue logger is doing).

Such lacks or inabilities should not deter anyone from flagging an issue, nor shame them either implicitly or explicitly, which is entirely possible with the phrasing of the above (and might well be, though it probably isn't, why it is't included as a given).

These are important things to keep in mind.